src/App.tsx

```diff
@@ -1,61 +1,39 @@
 import './App.scss';
+import { AddTodoForm } from './components/AddTodoForm/AddTodoForm';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { useState } from 'react';
+
+function getUserByID(userId: number) {
+  return usersFromServer.find(user => user.id === userId);
+}
+
+const array = todosFromServer.map(todo => ({
```

Comment:
`arr/array/el/elem/obj/object/item/res/result` are bad names, name should describe a variable